### ISSUE TITLE
action.yml image version is not properly updated

### ISSUE THREAD

**Issue by daltonv** [actions/container-prebuilt-action#14]
The way the repo is currently written the tag of image used in `action.yml` file isn't properly updated.

ie if I use the format the repo suggests:
```yaml
image: docker://ghcr.io/actions/container-prebuilt-action:latest
```

users of my action will always get the latest container image even if they specifically use an older version of the action. 

The only way to handle this would be for developers of the action to manually update the tagged version in the `action.yml` in every pr. That should at least be documented, but it really should be automated in someway. 

Thing is I'm not sure how the automation would work. Should there be a complicated `grep` `sed` script to update the tag in `action.yml` for you?

**Comment by ncalteen** [comment]
Sorry for the radio silence on this one! I was pulled into some other work but had some bandwidth to work on this today.

Regarding using automation to update the version in `action.yml`, I'm not personally a big fan of that approach because it can affect subsequent GitHub Actions runs for a PR. Mainly, if you use the default `${{ github.token }}` secret in a workflow run to update the contents, it will not trigger CI, linting, etc. to run again when the PR is updated. If you use a personal token or a GitHub App, it does run the risk of workflows triggering each other indefinitely.

I made some updates to the [`issue-ops/semver`](https://github.com/issue-ops/semver) action to support tracking semantic versioning in the `action.yml` file for container actions and created [this PR](https://github.com/actions/container-prebuilt-action/pull/19) to use that instead of a `.version` file for tracking version releases.

Basically, the process to update the action would be:

1. Create a branch
2. Make some changes to the action
3. Edit `action.yml` to increment the major/minor/patch version for the action
4. Create a PR

Once the PR is merged, the image is built and tagged with the new version from `action.yml`, and a new release is created for the action.
